.infrastructure/11_dns.tf

```
@@ -0,0 +1,4 @@
+module "dns_zone" {
+  source = "git::https://github.com/terraform-aws-modules/terraform-aws-route53.git//modules/zones?ref=bc63328714550fd903d2574b263833c9ce1c867e" # v2.11.0
```

```suggestion
  source = "git::github.com/terraform-aws-modules/terraform-aws-route53.git?ref=bc63328714550fd903d2574b263833c9ce1c867e"  # v2.11.0
```
I think this should work as well

---

no, it doesn't work if you remove the "//modules/zones" string: https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories

Error: Unsupported argument
│
│   on test.tf line 4, in module "zones":
│    4:   zones = {
│
│ An argument named "zones" is not expected here.

---

I didn't know that! thanks for sharing this! 🙏

---

of course. I had to do various tests and research to understand how to reference modules in subdirectories via sha